hummingbird/supported.py

```diff
@@ -0,0 +1,110 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+All operators, backends, and configurations settings supported in Hummingbird are registred here.
+"""
+from .exceptions import MissingConverter
+from ._utils import sklearn_installed, lightgbm_installed, xgboost_installed
+
+from xgboost import XGBClassifier, XGBRegressor
+from lightgbm import LGBMClassifier, LGBMRegressor
+
+
+def _build_sklearn_operator_list():
+    """
+    Put all suported Sklearn operators on a list.
+    """
+    if sklearn_installed():
+        # Tree-based models
+        from sklearn.ensemble import (
```

Comment:
Hmm...now we don't have one "clean" list anywhere.  And we link to this list in several places. Do you have suggestions?

Reply:
Ex: I simply dropped the list from https://github.com/microsoft/hummingbird/pull/49

Reply:
Yea. Do you think it is too much expecting users to read the supported file and find the operators? Perhaps I can list in the comment in supporter ops and backend. In this way it will also show in the documentation. WDYT?

Reply:
Yah reading through the file would be asking a lot, especially as the code grows around it.  I think we need a nice list somewhere.  A comment at the top could work. Or we could maintain one on the wiki...as long as we remember to maintain it ;)

Reply:
Agree that the problem is maintenance. What do you prefer? Eventually we can start with both and then ditch one of the two as we grow.

Reply:
I think  I like the wiki approach.I'll do that now.